CHANGELOG.md

```diff
@@ -4,7 +4,10 @@ Changelog
 2.2.0(ureleased)
 ---------------
 
-* **YYYY-MM-DD**:
+* **2015-02-26**:introduce PHPCS for style checks again
+* **2015-02-26**: Symfony 4.2 combustibility verified through tests, Symfony 4.3 allowed to break but on watch in tests
```

Comment:
what do you mean by combustibility? i would say "Symfony 4.2 support without warnings." and remove the rest, the changelog is not for us but for users of the bundle.